configure.ac

```diff
@@ -404,6 +404,7 @@ AS_IF([test x"$GCC" = "xyes"], [
 ])
 CFLAGS="$CFLAGS -D_GNU_SOURCE"
 CXXFLAGS="$CXXFLAGS -D_GNU_SOURCE -DPOINTERHOLDER_TRANSITION=0"
+CXXFLAGS="$CXXFLAGS -std=c++17"
```

Comment:
Principally correct, but please move it 2 lines down, under the comment, ass the comment belongs to line 406 (`CXXFLAGS="$CXXFLAGS -D_GNU_SOURCE -DPOINTERHOLDER_TRANSITION=0"`).